src/test/fuzz/data_stream.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2020 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <addrman.h>
+#include <net.h>
+#include <test/fuzz/FuzzedDataProvider.h>
+#include <test/fuzz/fuzz.h>
+#include <test/fuzz/util.h>
+
+#include <cstdint>
+#include <vector>
+
+void initialize_data_stream_addr_man()
+{
+    InitializeFuzzingContext();
+}
+
+FUZZ_TARGET_INIT(data_stream_addr_man, initialize_data_stream_addr_man)
+{
+    FuzzedDataProvider fuzzed_data_provider{buffer.data(), buffer.size()};
+    CDataStream data_stream = ConsumeDataStream(fuzzed_data_provider);
+    CAddrMan addr_man;
+    CAddrDB::Read(addr_man, data_stream);
+}
+
+void initialize_data_stream_net_message()
+{
+    InitializeFuzzingContext();
```

Comment:
Can you explain why it is required to boot up a full node to call a single constructor that only invokes `std::move`. Maybe the whole fuzz test isn't particularly useful, when it only calls `std::move`, which is already covered by other fuzz targets: https://marcofalke.github.io/btc_cov/fuzz.coverage/src/net.h.gcov.html#745

Reply:
Good catch. Removing :)